Sources/FPSStatusBarViewController.swift

```diff
@@ -86,7 +86,9 @@ extension FPSStatusBarViewController: FPSCounterDelegate {
     @objc func fpsCounter(_ counter: FPSCounter, didUpdateFramesPerSecond fps: Int) {
         let milliseconds = 1000 / max(fps, 1)
         self.label.text = "\(fps) FPS (\(milliseconds) milliseconds per frame)"
-
+        if FPSStatusBarViewController.statusBarWindow == UIApplication.shared.keyWindow {
```

Comment:
Could you please do `FPSStatusBarViewController.statusBarWindow.isKeyWindow` instead? This captures the intention more clearly.

Reply:
I've modified that，thx